rosidl_generator_py/CMakeLists.txt

```diff
@@ -23,6 +23,14 @@ if(BUILD_TESTING)
 
   find_package(test_interface_files REQUIRED)
 
+  find_package(python_cmake_module REQUIRED)
+  find_package(PythonExtra MODULE REQUIRED)
+
+  set(BUILDTYPE_PYTHON_EXECUTABLE "${PYTHON_EXECUTABLE}")
```

Comment:
This is a nit, but elsewhere where we do the `PYTHON_EXECUTABLE` dance, we typically call the variable "PYTHON_EXECUTABLE" (see https://github.com/ros2/rclpy/blob/c67e43a69a1580eeac4a90767e24ceeea31a298e/rclpy/CMakeLists.txt#L32-L37 for an example).  We may want to do the same here for consistency.

Reply:
I changed it away from `_PYTHON_EXECUTABLE` because the `rosidl_generate_interfaces()` call uses CMake code that [also uses that same variable name, but slightly differently](
https://github.com/ros2/rosidl_python/blob/master/rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake#L129-L132), and I didn't want to have to think about whether this was correct with that or not.